galata/extension/src/global.ts

```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
```

Comment:
This one has undated Bloomberg copyright. Not sure what to do here.

Reply:
Well, moving the old Bloomberg copyright one line down pleases the header linter.